config/initializers/assets.rb

```diff
@@ -1,5 +1,10 @@
 # Be sure to restart your server when you modify this file.
 
+Rails.application.config.assets.webpack_manifest =
+  if File.exist?(Rails.root.join('public', 'assets', 'manifest.json'))
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.


<!-- Comment by [SideCI](https://sideci.com) -->